src/shapes/object.class.js

```diff
@@ -1954,6 +1954,7 @@
     dispose: function () {
       if (fabric.runningAnimations) {
         fabric.runningAnimations.cancelByTarget(this);
+        this.clipPath && fabric.runningAnimations.cancelByTarget(this.clipPath);
```

Comment:
let's not do that.
Because otherwise we should dispose the clipPath too, while we decided that we can't decide for the developer.

Reply:
Alright